explainer.html

```diff
@@ -0,0 +1,179 @@
+                    </p>
+                    <p>
+                        The term “named graphs” is also used in literature or in applications.
+                    </p>
+                </dd>
+
+                <dt>Identical RDF Datasets</dt>
+                <dd>
+                    <p>
+                        <span class='rdf'>R = R‘&nbsp;</span> denotes two <em>identical</em> RDF Datasets.
+                    </p>
+                    <p>
+                        Identical RDF Datasets consists of the same <a href="https://www.w3.org/TR/rdf11-concepts/#section-rdf-graph">RDF Graphs</a> (where “the same” means they are the same set of triples) with the same <a href="https://www.w3.org/TR/rdf11-concepts/#section-dataset">Graph Names</a>.
+                    </p>
+                </dd>
+
+                <dt>Isomorphic RDF Datasets</dt>
+                <dd>
+                    <p>
+                        <span class='rdf'>R ≈ R‘&nbsp;</span> denotes two <a href='https://www.w3.org/TR/rdf11-concepts/#section-dataset-isomorphism'><em>isomorphic</em></a> RDF Datasets.
+                    </p>
+                    <p>
+                        The structure of two isomorphic Datasets are identical: URIs and literals are all character-wise identical, and it is possible to relabel the blank nodes of <span class='rdf'>R</span> to the blank nodes of <span class='rdf'>R‘&nbsp;</span> without changing the topology of the graph.
+                    </p>
+                </dd>
+
+                <dt id="canonical_form">Canonical Form</dt>
+                <dd>
+                    <p>
+                        For an RDF Dataset <span class='rdf'>R</span>, a <em>canonical form</em> of <span class='rdf'>R</span> is a dataset <span class='rdf'>R<sub>C</sub></span> such that <span class='rdf'>R<sub>C</sub> = R’<sub>C</sub>&nbsp;</span> if and only if <span class='rdf'>R ≈ R’</span>.
+                    </p>
+                </dd>
+
+                <dt id='canonicalization'>Canonicalization</dt>
+                <dd>
+                    <p>
+                        Canonicalization is a function <span class='rdf'>C&nbsp;</span> on RDF Datasets such that <span class='rdf'>C(R)</span> is a Canonical Form for <span class='rdf'>R</span>
+                    </p>
+                    <p>
+                        Definition of a canonicalization function means, in practice, to define a deterministic re-labeling of <em>all</em> blank nodes of an RDF Dataset without changing the structure of the graph.
+                    </p>
+                </dd>
+            </dl>
+        </section>
+
+        <p>
+            Defining a proper canonicalization function was an unsolved mathematical problems for several year. By now, however, two solutions exists:
+        </p>
+        
+        <ol>
+            <li>
+                The algorithms defined Aidan Hogan in [[aidan-2017]], vetted through the scholarly peer review process.
```

Comment:
I'm concerned that this comes across as "one of these has been peer reviewed and the other one hasn't"... which isn't true. One of them can trace its lineage back in time to some of the more formal papers, that's true, but we shouldn't be giving one of them more weight than the other because of that. A solution is a solution, no matter where it came from.

Reply:
@msporny, this is obviously not my intention. What we must have, to amend this text properly, is a public, stable address for the paper of Dave and Rachel, plus a separate pointer to the review results. From our point of view, this would then be in par with the other paper. As soon as we have those pointers we can/should update the text.

Flagging this: this must be done _before_ we even move this to the W3C process. In other words, ASAP.